packages/cli/src/commands/epochs/status.ts

```diff
@@ -0,0 +1,78 @@
+import { ux } from '@oclif/core'
+import { PublicClient } from 'viem'
+import { BaseCommand } from '../../base'
+import { getEpochManagerContract } from '../../packages-to-be/contracts'
+
+export default class EpochStatus extends BaseCommand {
+  static description = 'View epoch info.'
+
+  static flags = {
+    node: BaseCommand.flags.node,
+    ...(ux.table.flags() as object),
+  }
+
+  static aliases: string[] = ['epochs:current']
+
+  static args = {}
+
+  static examples = []
+
+  async run() {
+    const res = await this.parse(EpochStatus)
+
+    const client = await this.getPublicClient()
+    ux.action.start('Fetching current epoch information')
+
+    const epochManager = await getEpochManagerContract(client as unknown as PublicClient)
+
+    const results = await Promise.allSettled([
+      epochManager.read.getCurrentEpoch(),
+      epochManager.read.getCurrentEpochNumber(),
+      epochManager.read.isEpochProcessingStarted(),
+      epochManager.read.isOnEpochProcess(),
+      epochManager.read.isIndividualProcessing(),
+      epochManager.read.isTimeForNextEpoch(),
+    ])
+    ux.action.stop('Done\n')
+
+    const [
+      currentEpoch,
+      currentEpochNumber,
+      isEpochProcessingStarted,
+      isOnEpochProcess,
+      isIndividualProcessing,
+      isTimeForNextEpoch,
+    ] = results.map((result) => (result.status === 'fulfilled' ? result.value : result.reason))
+
+    const [firstBlock, lastBlock, startTimestamp] = currentEpoch
```

Comment:
The `lastBlock` of an epoch is not known until the epoch finishes. this means that the `lastBlock` of the current epoch will always return 0

Reply:
Good to know!